libmscore/edit.cpp

```diff
@@ -1728,6 +1728,10 @@ void Score::deleteItem(Element* el)
                               m->undoChangeProperty(Pid::REPEAT_START, false);
                         else if (bl->barLineType() == BarLineType::END_REPEAT)
                               m->undoChangeProperty(Pid::REPEAT_END, false);
+                        else if (bl->barLineType() == BarLineType::END_START_REPEAT) {
```

Comment:
Should not be needed if this element is never added to the score.

Reply:
yes, need to delete both

Reply:
Maybe, that's not really clear.  Maybe you only *want* to delete one side.  That could well be the more common case, actually.